dev-server/documents/html/parent-frame.mustache

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Annotatable iframe test</title>
+<style>
+  body {
+    font-family: sans-serif;
+  }
+  iframe {
+    width: 100%;
+    height: 400px;
+    resize: both;
+    overflow: auto;
+  }
+</style>
+<script>
+  // Make the iframe annotatable after the sidebar has been created.
+  // This test if injected client in the iframe is able to discover the sidebar iframe
+  setTimeout(()=> document.querySelector('iframe').setAttribute('enable-annotation', ''), 5000)
```

Comment:
This timeout seems quite long. Maybe make it a little shorter (3s or so)?

Reply:
Sure